src/sentry/static/sentry/app/components/sidebar/index.jsx

```diff
@@ -192,11 +194,19 @@ const Sidebar = React.createClass({
           currentPanel={this.state.currentPanel}
           onShowPanel={()=>this.togglePanel('statusupdate')}
           hidePanel={()=>this.hidePanel()} />
-        <li>
-          <a title="Support" href={`/organizations/${org.slug}/support/`}>
```

Comment:
Can we just simplify this to just:

```javascript
href={config.isOnpremise ? 'https://forum.sentry.io/' : `/organizations/${org.slug}/support/`}
```

Rather than copy/pasting all the DOM.